languages/go/exercises/.docs/cli.md

```diff
@@ -0,0 +1,17 @@
+# CLI (Command Line Interface)
```

Comment:
The location for this file should change to `exercises/shared/.docs/cli.md`. See #54.